gui/default/syncthing/settings/settingsModalView.html

```diff
@@ -172,6 +172,13 @@
               </div>
             </div>
             <div class="col-md-6">
+              <div ng-if="tmpGUI.address.startsWith('/') || tmpGUI.address.startsWith('unix://') || tmpGUI.address.startsWith('unixs://')" class="form-group" ng-class="{'has-error': settingsEditor.UnixSocketPermissions.$invalid && settingsEditor.UnixSocketPermissions.$dirty}">
```

Comment:
(entirely optional) That's long enough that it could warrant a `isUnix(address)` util function.

Reply:
Done.